contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -942,10 +942,10 @@ get_authid_user_ext_physical_name(const char *db_name, const char *login)
 {
 	Relation	bbf_authid_user_ext_rel;
```

Comment:
This same APIs in being used in has_dbaccess() system function. Do we have enough tests covering grant/revoke connect to user and then checking the has_dbaccess()?